### ISSUE TITLE
Weird 'ghost' windows appearing on every workspace

### ISSUE THREAD

**Issue by dextdc** [paperwm/PaperWM#784]
I get these two mysterious blank windows appearing on every workspace.

I uninstalled PaperWM completely in Extensions-manager, then re-installed, trying to get rid of them. 

Are these normal? My PaperWM has been freezing the entire Gnome alot, and I think these are the cause.

![Screenshot from 2024-02-17 23-56-56](https://github.com/paperwm/PaperWM/assets/153247637/ff50e54f-3106-42a0-a629-f5e6da23f2c4)

![Screenshot from 2024-02-18 00-00-13](https://github.com/paperwm/PaperWM/assets/153247637/e7a65767-79eb-4436-8a5a-c0f96c6b2bc7)

**Comment by dextdc** [comment]
I have been able to trace the cause of this to the Gnome-Shell extension Desktop Icons NG (DING). It was built into the standard Ubuntu 22.04 I am using. 

After disabling DING, the two ghosts windows go away. The ghost windows appear to cause PaperWM to crash when trying to recalculate windows due to these ghost windows appearing in every workspace. I had 3 crashes within 5 minutes of rebooting due to these Ghost windows -- just load all about 5-8 browser windows and move them around, and then can generate the crash.

This all started when I attempted to install Dash to Dock over the built-in Ubuntu Dock. This appears to have caused DING issues which then cascaded into these two Ghost windows appearing.
 
Note that I am only using PaperWM gnome-extension, and the 3 built-in Ubuntu 22.04 gnome-extensions (DING, Ubuntu AppIndicators, Ubuntu Dock).

**Comment by Lythenas** [comment]
The issues with DING are known. Although I can't find if it is documented anywhere :(

I think it is unlikely that we can make DING work with PaperWM properly. And personally I don't really find it useful.

Maybe it would be useful to detect if conflicting extensions are enabled and give a warning. What do you think? @jtaala 

In the mean time I will add a note about this to the readme.

**Comment by jtaala** [comment]
> Maybe it would be useful to detect if conflicting extensions are enabled and give a warning. What do you think? @jtaala

I think this is a very good idea (as well as adding these to the README).  I've been tempted to detect these extensions and give a warning and not run PaperWM until they are disabled (although that might be a bit overkill...).

I'm thinking of DING and https://github.com/paperwm/PaperWM/issues/763 specifically.

**Comment by jtaala** [comment]
> I think it is unlikely that we can make DING work with PaperWM properly. And personally I don't really find it useful.

Yes, I wouldn't fix/address this specifically for DING (don't use it etc.).  However, if someone wants to work on compatibility for it, I'd look at accepting a PR for this.
